.gitignore

```diff
@@ -45,6 +45,7 @@ ipch/
 *.VC.opendb
 .vs/
 .vscode/
+.idea/
```

Comment:
If you need `.idea/` in your .gitignore, consider adding it to a personal global `.gitignore` file instead. https://help.github.com/articles/ignoring-files/#create-a-global-gitignore